### PR TITLE
Add ssh-tunnel-on-gce tutorial

### PR DESCRIPTION
**Pull request by ahmetb**
cc: @jimtravis feel free to patch my branch directly

Signed-off-by: Ahmet Alp Balkan <ahmetb@google.com>

### CODE REVIEW

File: tutorials/ssh-tunnel-on-gce.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+network might be enforcing:
+
+```
++--------+       inspection      +----------+
+| Laptop |---------------------->| Internet |
++--------+       firewalls       +----------+
+```
+
+Not only your ISP, other parties who can get in the middle can block your
+access to websites, they can also inspect and modify the contents of your
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: Start a new sentence: "...access to websites.` `They can..."

File: tutorials/ssh-tunnel-on-gce.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+---
+
+
+In this tutorial, you will explore how you can use [Google Compute Engine][gce]
+Linux instances to route all your local network traffic with an encrypted SSH
+tunnel.
+
+This tutorial is not meant to be used for routing your server application
+traffic, but rather to set a VPN-like proxy on your laptop or workstation to
+bypass certain network limits, such as censorship, and browse internet
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: add "the", capitalize Internet: "...and browse `the Internet` privately."

File: tutorials/ssh-tunnel-on-gce.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+### Set up the SSH tunnel
+
+First of all, you need a compute instance to route all your traffic through it.
+If you have an existing instance, you can use it, or create a new a compute
+instance named `tunnel` from Google Cloud Platform Console or from `gcloud`:
+
+    gcloud compute instances create --zone us-west1-a tunnel
+    
+Start an SSH tunnel on your machine on a local port, such as 5000 that connects
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: add comma: "...such as 5000`,` that connects..."

File: tutorials/ssh-tunnel-on-gce.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+to a GCE instance on its SSH port (which is 22):
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: use parentheses sparingly: "...on its SSH port, which is 22:"

File: tutorials/ssh-tunnel-on-gce.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+    gcloud instances ssh --zone us-west1-a tunnel -- -N -p 22 -D localhost:5000
+
+This command works out of the box on mac OS, Windows and Linux, and starts an
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: no space in [macOS](https://en.wikipedia.org/wiki/MacOS)

File: tutorials/ssh-tunnel-on-gce.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+applications, such as browsers, might have their own separate proxy settings.
+
+Once the SSH tunnel is started using the command above, your proxy host is
+`localhost` and port is `5000`.
+
+Here are some useful links to configure the proxy in various platforms:
+
+- **Windows:** Follow the “using the SOCKS proxy” section in [this article][win]
+  [[mirror][win-a]] to enable it on Internet Explorer, Edge and Firefox.
+- **mac OS:** System Preferences &rarr; Network &rarr; Advanced &rarr; Proxies
```
Comment (chenopis) [inline tutorials/ssh-tunnel-on-gce.md]: same as above, no space in macOS

File: tutorials/ssh-tunnel-on-gce.md (hunk 2 above)
Comment (ahmetb) [inline tutorials/ssh-tunnel-on-gce.md]: @chenopis I have mixed feelings about this: http://www.theverge.com/2016/4/2/11352744/ap-style-guide-will-no-longer-capitalize-internet

**Comment by ahmetb** [comment]
@chenopis fixed the comments except capitalizing internet. 😇

**Comment by chenopis** [comment]
@ahmetb Oh man, I think I'm showing my age by capitalizing Internet. :(

**Pull request merged**